CHANGELOG.md

```diff
@@ -0,0 +1,241 @@
+# Change Log for AWS IoT Embedded C SDK
+
+## 202009.00 (September 14, 2020)
+
+- The MQTT, JSON and AWS IoT Shadow service client libraries have been moved to the following individual repositories:
+   - [FreeRTOS/coreMQTT](https://github.com/FreeRTOS/coreMQTT) for MQTT client library
+   - [FreeRTOS/coreJSON](https://github.com/FreeRTOS/JSON) for JSON parser library
+   - [aws/device-shadow-for-aws-iot-embedded-sdk](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk) for AWS IoT Shadow service client library.
+- The above library folders in `libraries/` folder has been replaced with submodule to the individual repositories.
+
+## [3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/releases/tag/v3.0.1) (May 10, 2018)
```

Comment:
We would need to also mention about the tag 3.1.0 which was the latest content in master of V3 CSDK, which was unreleased.

Reply:
Good point, have added text for the tag